src/vsphere_cpi/spec/unit/cloud/vsphere/soap_stub_spec.rb

```diff
@@ -8,21 +8,27 @@
   let(:retryable_adapter) { instance_double(VSphereCloud::SdkHelpers::RetryableStubAdapter) }
 
   describe '#create' do
-    it 'returns the SDK Soap Adapter' do
-      expect(VimSdk::Soap::StubAdapter).to receive(:new)
-        .with(
-          'https://some-host/sdk/vimService',
-          'vim.version.version12',
-          http_client,
-        )
-        .and_return(base_adapter)
-      expect(VSphereCloud::SdkHelpers::RetryableStubAdapter).to receive(:new)
-       .with(
-         base_adapter,
-       )
-       .and_return(retryable_adapter)
-
-      expect(soap_stub.create).to eq(retryable_adapter)
+    before(:all) do
```

Comment:
Should be `before(:each)` and `after(:each)` probably. Also this is a perfect time to use `around` rather than `before` and `after`.

Reply:
I will use `around` and fix spacing in the test. I am changin VC version in the tests, so I want to preserve and set the old value **once** for `all` tests